released/packages/coq-fourcolor-reals/coq-fourcolor-reals.1.4.0/opam

```diff
@@ -14,8 +14,9 @@ with proofs of properties such as categoricity."""
 build: [make "-C" "theories/reals" "-j%{jobs}%"]
 install: [make "-C" "theories/reals" "install"]
 depends: [
-  "coq" {(>= "8.16" & < "8.21~") | (= "dev")}
-  "coq-mathcomp-ssreflect" {(>= "2.1.0" & < "2.4~") | (= "dev")}
+  ("coq" {>= "8.16" & < "8.21~"} |
+   "rocq-core"{ (>= "9.0" & < "9.1~") | (= "dev")0})
+  "coq-mathcomp-ssreflect" {(>= "2.1.0" & < "2.5~") | (= "dev")}
```

Comment:
Are we sure that fourcolor 1.4.0 compiles on MC 2.4.0? My last nix attempt seems to suggest the contrary: https://github.com/rocq-community/coq-nix-toolbox/pull/340

Reply:
Yes, I made a mistake: I tested the head version on rocq-community, not 1.4.0!